cluster-provision/k8s/1.26/provision.sh

```diff
@@ -3,16 +3,7 @@
 set -ex
 
 function getKubernetesClosestStableVersion() {
-  kubernetes_version=$version
-  packages_version=$kubernetes_version
-  if [[ $kubernetes_version == *"alpha"* ]] || [[ $kubernetes_version == *"beta"* ]] || [[ $kubernetes_version == *"rc"* ]]; then
-    kubernetes_minor_version=$(echo $kubernetes_version | cut -d. -f2)
-    packages_major_version=$(echo $kubernetes_version | cut -d. -f1)
-    packages_minor_version=$((kubernetes_minor_version-1))
-    packages_patch_version=0
-    packages_version="$packages_major_version.$packages_minor_version.$packages_patch_version"
-  fi
-  echo $packages_version
+  echo "$(curl --fail -L https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//')"
```

Comment:
I think that it might be a problem when the stable version will be higher than 1.26

Reply:
Can you elaborate, please?

I see that the stable is always lower than the latest:
```
$ curl --fail -L https://storage.googleapis.com/kubernetes-release/release/latest.txt | sed 's/^v//'
1.27.0-alpha.0%
$ curl --fail -L https://storage.googleapis.com/kubernetes-release/release/stable.txt | sed 's/^v//'
1.26.0%
```

Reply:
In the future when 1.28 will be release the 1.26 prisoner will try to use kubeadm,kubelet and kubectl with 1.28 version
and according to Kubernetes docs this is not safe:
 https://kubernetes.io/releases/version-skew-policy/#kubelet
https://kubernetes.io/releases/version-skew-policy/#kubectl
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#kubeadm-s-skew-against-the-kubernetes-version

Reply:
Meh, you are right! Thanks!

Reply:
@Barakmor1 I've updated `getKubernetesClosestStableVersion` such that it fetches the latest stable (instead of the 0) from the previous release. I think that brings us closer to what we want. Sorry for being so nitty about this :slightly_smiling_face: